src/services/database/handlers/seeddatabase.ts

```diff
@@ -0,0 +1,24 @@
+import { send, SUCCESS, FAILED } from "cfn-response-async";
+import { putItem } from "../../../libs";
+import * as items from "../items.json";
+
+exports.handler = async function (event: any, context: any) {
+  console.log("Request:", JSON.stringify(event, undefined, 2));
+  const responseData = {};
+  let responseStatus: any = SUCCESS;
+  try {
+    for (let i = 0; i < items.length; i++) {
+      console.log(items[i].recordId);
+      console.log(items[i].state);
+      await putItem({
+        tableName: event.ResourceProperties.DynamoTableName,
+        item: { PK: items[i].recordId, SK: items[i].state },
+      });
+    }
```

Comment:
```suggestion
    for await (const item of items) {
      console.log({item});
      putItem({
        tableName: event.ResourceProperties.DynamoTableName,
        item: { PK: item.recordId, SK: item.state },
      });
    }
```

might clean it up by doing somehting like this